packages/swapper/src/swappers/JupiterSwapper/swapperApi/getTradeQuote.ts

```diff
@@ -163,6 +232,10 @@ export const getTradeQuote = async (
     convertJupiterInstruction(swapResponse.swapInstruction),
   ]
 
+  if (feeAccountInstruction && affiliateBps !== '0') {
```

Comment:
Don't create a referral account if you don't have to pay fees, with this we ensure only big trades are creating referral accounts if they don't exists